public/index.html

```diff
@@ -146,26 +146,28 @@
       <div class="logo_text1">Sweet bear</div>
       <div class="logo_text2">Bakery blog</div>
     </div>
-    <section class="subscribe input-field">
-      <input
-        type="text"
-        class="input-field__input"
-        placeholder="Enter your e-mail..."
-      />
-      <!-- letetr icon image -->
-      <svg
-        class="input-field__icon input-field__icon--letter"
-        width="40"
-        height="29"
-        viewBox="0 0 40 29"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M39.6554 0.910205L20.4101 15.1408C20.1718 15.3172 19.8451 15.3172 19.5978 15.1497L0.149944 1.2896C0.0529528 1.52791 0 1.78408 0 2.0575V26.8905C0 28.0294 0.92711 28.9562 2.0657 28.9562H37.9339C39.0728 28.9562 39.9996 28.0291 39.9996 26.8905V2.06617C40.0085 1.6422 39.8761 1.2361 39.6553 0.909692L39.6554 0.910205ZM1.06783 0.256838L19.9955 13.7371L38.4723 0.0708171C38.3045 0.0267876 38.1191 0.000308502 37.9337 0.000308502L2.07345 0C1.71167 0 1.36744 0.0883679 1.0672 0.256173L1.06783 0.256838Z"
+    <div class="subscribe">
+      <div class="input-field">
+        <input
+          type="text"
+          class="input-field input-field__input"
```

Comment:
Why we have double input-filed class, it should be only on parent div
```suggestion
          class="input-field__input"
```

Reply:
done